messaginginapp/src/main/java/io/customer/messaginginapp/ModuleMessagingInApp.kt

```diff
@@ -0,0 +1,74 @@
+package io.customer.messaginginapp
+
+import android.app.Application
+import io.customer.messaginginapp.di.gistProvider
+import io.customer.sdk.CustomerIO
+import io.customer.sdk.CustomerIOModule
+import io.customer.sdk.data.request.MetricEvent
+import io.customer.sdk.di.CustomerIOComponent
+import io.customer.sdk.hooks.HooksManager
+import io.customer.sdk.hooks.ModuleHook
+import io.customer.sdk.hooks.ModuleHookProvider
+import io.customer.sdk.repository.TrackRepository
+
+class ModuleMessagingInApp internal constructor(
+    private val overrideDiGraph: CustomerIOComponent?,
+    private val organizationId: String
+) : CustomerIOModule {
+
+    constructor(organizationId: String) : this(
```

Comment:
```suggestion
    constructor(gistOrganizationId: String) : this(
```

This might be a question for product - because CIO does not use the term "organization id" anywhere in the product, I think that just saying "organization id" will confuse customers where they don't understand what we are asking of them. Instead, if we label it as `gistOrganizationId` customers should understand better what we are asking.

Reply:
I asked product about this to see if they have a preference. This is an easy fix to change later.

Reply:
Product decided to keep the naming as it is in this PR.